pkg/virt-handler/cmd-client/client.go

```diff
@@ -98,11 +105,19 @@ const (
 	longTimeout  time.Duration = 20 * time.Second
 )
 
-func ListAllSockets(baseDir string) ([]string, error) {
+func SetLegacyBaseDir(baseDir string) {
```

Comment:
Can we introduce a struct?

Reply:
i'd rather not. This isn't a value that's dynamic. It's (optionally) be set once at initialization, then everything that consumes the client doesn't have to keep up with whatever the client base directory is. If it's not set, a default value is used. We've been using this pattern for a bit other places as well.

Reply:
I know that we used that pattern on more places. I don't think it is good that we do. It is much clearer to have clear dependency boundaries from my perspective (for instance testing is much clearer). I would prefer if we don't do that in the future anymore.